### ISSUE TITLE
Changelog warning in pip install.

### ISSUE THREAD

**Issue by xtoddx** [oracle/oci-cli#27]
```
(venv)[vagrant@ol7 ~]$ pip install oci-cli
Downloading/unpacking oci-cli
  Downloading oci-cli-2.4.11.tar.gz (169kB): 169kB downloaded
  Running setup.py egg_info for package oci-cli

    warning: no files found matching 'CHANGELOG.md'
```

I see that MANIFEST.in has CHANGELOG.md instead of CHANGELOG.rst, which may be the root cause?

**Comment by nathan-vu** [comment]
Yep, that's probably it. Will be fixed for our next release.

**Comment by mross22** [comment]
Fixed in release 2.4.12
